doc/tutorials/transformer/masked_modelling.ipynb

```diff
@@ -0,0 +1,277 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        EXTERNAL_HOST_NAME = "192.168.124.93"
</pre><p>i suggest changing these to somethin like<code> "EXTERNAL_HOST_NAME = "&lt;your_host_server&gt;"		# change this		</code></p><p>also for the user and password. i think having these filled out in a tutorial looks weird. even if you plan to chnage it to the trust store later</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>